cms/templates/js/course-video-settings-update-org-credentials-footer.underscore

```diff
@@ -1,4 +1,8 @@
+<button class='button-link action-cancel' aria-describedby='cancel-button-text'>
+    <%- gettext('Reset') %>
```

Comment:
@srpearce Do you think we need to changes this text for `Reset` button? This button actually resets changes back to what are saved in database / initial state (much like Discard Change). Do you have any other suggestions, or we good here ?

Reply:
Does "Cancel" work?

Reply:
"Cancel" may look a bit confusing to user i.e they may think cancel button to close the this Settings pane immediately.